service/spi/src/main/java/gov/va/vro/service/spi/model/SimpleClaim.java

```diff
@@ -0,0 +1,17 @@
+package gov.va.vro.service.spi.model;
+
+import lombok.*;
+
+import java.util.List;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@ToString(includeFieldNames = true)
+public class SimpleClaim {
```

Comment:
There is no need for a new POJO  class. Use ClaimMapper to convert from Entity to Claim. Then convert from Claim to ClaimInfo